files/validator.py

```diff
@@ -425,24 +416,25 @@ def _validate_duplicate_keys(self, var_file):
         clusters = 'clusters'
         domains = 'domains'
         roles = 'roles'
-        aff_group = 'aff_groups'
-        aff_label = 'aff_labels'
+        aff_groups = 'aff_groups'
+        aff_labels = 'aff_labels'
         network = 'network'
         key1 = 'primary_name'
         key2 = 'secondary_name'
+        dr_primary_name = 'dr_primary_name'
```

Comment:
used only once, why do we need it as a variable?

Reply:
> used only once, why do we need it as a variable?

1) For consistency with others (although I agree that the existing variables are used at least twice).
2) To be able to fit "[domains, self.domain_map, dr_primary_name, dr_secondary_name]," into 1 line.

But that's not critical, I can revert this part.
I thought that it improves the readability, especially that this method already defines 8 similar string variables, so 2 new ones match the style and don't look ugly :)

Reply:
Not that critical, leave it as is